x/txfees/types/options.go

```diff
@@ -85,3 +97,22 @@ func parseDecFromConfig(opts servertypes.AppOptions, optName string, defaultValu
 	}
 	return value
 }
+
+func parseBoolFromConfig(opts servertypes.AppOptions, optName string, defaultValue bool) bool {
+	valueInterface := opts.Get("osmosis-mempool." + optName)
+	value := defaultValue
+	if valueInterface != nil {
+		valueStr, ok := valueInterface.(string)
+		if !ok {
+			panic("invalidly configured osmosis-mempool." + optName)
+		}
+		valueStr = strings.ToLower(valueStr)
+
+		if valueStr == "true" {
+			return true
+		} else if valueStr == "false" {
+			return false
+		}
```

Comment:
Why not using `strconv.ParseBool`. Are we really forbidding usage of some boolean representations they are using? 

Anyway this could be simplified with [strings.EqualFold](https://pkg.go.dev/strings#EqualFold) (+ it handles more case than `toLower`)

Reply:
Thats a good plan. This was purely what my AI editor suggested haha